.github/workflows/test-cloud-platforms.yml

```diff
@@ -0,0 +1,149 @@
+name: Test cloud platforms
+env:
+  REGISTRY: ghcr.io
+  CONTAINER_REPO: ${{ github.repository }}
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+on:
+  schedule:
+    # Each week day at 9:30 UTC (so 3:30 MDT and 2:30 MST).
+    - cron: "30 9 * * MON-FRI"
```

Comment:
Why should we schedule this while ARO is running when merging on `main` or making a release? The main problem I see with this scheduling is that it will not run on the tagged commit before making a release.

Reply:
> I am concerned about the time it takes to setup the cluster and then running the tests.
> But running them only on main should be fine.
>
Running the integration tests on ARO takes around 10 minutes, and according to [this](https://github.com/kinvolk/inspektor-gadget/actions/runs/3021525878), setting up the AKS cluster + running the integration tests took around 12 minutes. I think two additional minutes on the CI, only when merging or releasing, is ok.

> In this case, I think we do not need to have a dedicated file and can add the job to the workflow file we already have.
>
Yeah, in that case, we can keep everything on one single workflow, and we don't duplicate the compilation of `kubectl-gadget`. Thus, this PR should become even simpler.